backends/dpdk/dpdk.def

```diff
@@ -1,5 +1,6 @@
 interface IDPDKNode {
     virtual std::ostream& toSpec(std::ostream& out) const = 0;
+    static ordered_map<cstring, cstring> origNameMap;
```

Comment:
Do you have to use a name map on every backend node? Can you move it to the ShortenTokenLength pass and apply that pass on the backend IR tree before codegen?

Reply:
done